app/build.gradle

```diff
@@ -97,6 +97,9 @@ dependencies {
     implementation 'androidx.lifecycle:lifecycle-extensions:2.2.0'
     implementation 'androidx.navigation:navigation-fragment-ktx:2.4.1'
     implementation 'androidx.navigation:navigation-ui-ktx:2.4.1'
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.4")
```

Comment:
Remove this kotlin coroutines library as it appears twice in the build.gradle file. Most likely a merge issue